package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "pretty-json-log",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

```suggestion
  "version": "1.4.1",
```

CI should auto bump this.